api/v1alpha1/servicedefaults_types.go

```diff
@@ -54,9 +54,8 @@ func init() {
 // ToConsul converts the entry into it's Consul equivalent struct.
 func (s *ServiceDefaults) ToConsul() *capi.ServiceConfigEntry {
 	return &capi.ServiceConfigEntry{
-		Kind: capi.ServiceDefaults,
-		Name: s.Name,
-		//Namespace: s.Namespace, // todo: don't set this unless enterprise
```

Comment:
We don't have access to the namespace config here so we can't determine the namespace. This will be done in the controller when making the api calls to consul.